rtt_roscomm/src/passthrough_callback_queue.cpp

```diff
@@ -0,0 +1,24 @@
+
+#include "rtt_roscomm/passthrough_callback_queue.hpp"
+
+namespace rtt_roscomm {
+
+PassthroughCallbackQueue::PassthroughCallbackQueue()
+        {}
+
+void PassthroughCallbackQueue::addCallback(
+    const ros::CallbackInterfacePtr &callback,
+    uint64_t owner_id)
+{
+  // Call CallbackInterface::CallResult SubscriptionQueue::call()
+  if (ros::CallbackInterface::TryAgain == callback->call()) {
```

Comment:
nit: Make the call more explicit here? For example
```suggestion
  const ros::CallbackInterface::CallResult result = callback->call();
  if (ros::CallbackInterface::TryAgain == result) {
```

That would be obvious without the need for the additional comment.